src/schema/formats.js

```diff
@@ -109,3 +109,62 @@ export function imageDataOrStrictRelativeUrl(value) {
 }
 
 export const isUnresolvedRelativeUrl = isStrictRelativeUrl;
+
+export function manifestShortcutKey(value) {
+  // Partially taken from Firefox directly via
```

Comment:
I guess that we should also mention here how we will have to handle updates on the validation of the shortcut keys:

now that the allowed keys are not part of the schema files, "changes to the allowed shortcut keys" becomes something that we need to check manually when we update the schema files, by looking if there are any changes to the validation code on the Firefox side and be sure to replicate them here.

Reply:
Done in dc639e2c